libraries/AP_OpticalFlow/AP_OpticalFlow_Pixart.cpp

```diff
@@ -79,7 +83,7 @@ extern const AP_HAL::HAL& hal;
 AP_OpticalFlow_Pixart::AP_OpticalFlow_Pixart(OpticalFlow &_frontend) :
     OpticalFlow_backend(_frontend)
 {
-    _dev = std::move(hal.spi->get_device("external0m3"));
+    _dev = std::move(hal.spi->get_device("pixartflow"));
```

Comment:
my apologies, that's what it is on my test board, but this PR is driver only, not including the new spi devices for this board. I've fixed.